### PR TITLE
chore: Change internal and external links target

### PR DESCRIPTION
**Pull request by juliajforesti**
[DSAR-478](https://rocketchat.atlassian.net/browse/DSAR-478)

## Proposed changes (including videos or screenshots)

- internal links should open on same page `target="_self"`
image

- external links should open on new tab `target="_blank"`
image

## Further comments

[DSAR-478]: https://rocketchat.atlassian.net/browse/DSAR-478?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/meteor/client/components/MarkdownText.tsx [hunk 1]
```diff
@@ -14,6 +15,27 @@ type MarkdownTextParams = {
 	withTruncatedText: boolean;
 } & ComponentProps<typeof Box>;
 
+const getBaseURI = (): string => {
```
Comment (tassoevan) [inline apps/meteor/client/components/MarkdownText.tsx:18]: `question` Why is this function duplicating the IIFE at `apps/meteor/client/lib/baseURI.ts`?

File: packages/gazzodown/src/elements/LinkSpan.tsx [hunk 2]
```diff
@@ -67,7 +69,7 @@ const LinkSpan = ({ href, label }: LinkSpanProps): ReactElement => {
 	}
 
 	return (
-		<a href={href} title={href}>
+		<a href={href} title={`${t('Go_to_href', { href: href.replace(getBaseURI(), '') })}`}>
```
Comment (tassoevan) [inline packages/gazzodown/src/elements/LinkSpan.tsx:72]:
```suggestion
		<a href={href} title={t('Go_to_href', { href: href.replace(getBaseURI(), '') })}>
```

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: packages/gazzodown/src/elements/LinkSpan.tsx (hunk 2 above)
Comment (juliajforesti) [reply packages/gazzodown/src/elements/LinkSpan.tsx:72]: ops, thanks

File: apps/meteor/client/components/MarkdownText.tsx (hunk 1 above)
Comment (juliajforesti) [reply apps/meteor/client/components/MarkdownText.tsx:18]: fair question :) where do you suggest I should export it from?

**Review by tassoevan — APPROVED** [review]

**Pull request merged**